app/lib/core/constants/indonesiaProvince.dart

```diff
@@ -0,0 +1,57 @@
+class IndonesiaProvince {
+  String provinceName;
+  String valueProvince;
+  bool isActive;
+
+  IndonesiaProvince({
+    required this.provinceName,
+    required this.valueProvince,
+    this.isActive = true,
+  });
+
+  String userAsString() {
+    return this.provinceName;
+  }
+}
+
+List<IndonesiaProvince> provinceList = [
+  IndonesiaProvince(provinceName: "Bali", valueProvince: "bali", isActive: true,),
```

Comment:
hmm, why we need a different value in `valueProvince`?
"Nusa Tenggara Timur" as is should be clear enough

Reply:
idem